media/stump-radio/README

```diff
@@ -0,0 +1,73 @@
+Minimalistic mplayer-based radio for the window manager StumpWM
+---------------------------------------------------------------
+
+I just want to be able to switch on some music by a press of a button.
+
+Depends on mplayer which is started via SB-EXT:RUN-PROGRAM.
+
+Hacked by Max-Gerd Retzlaff.
+
+
+loading and configuration
+-------------------------
+
+You can automatically load the module by adding this to your
+~/.stumpwm.d/init.lisp:
```

Comment:
See my pull request here https://github.com/m-g-r/stumpwm-contrib/pull/1 for ideas for improvements to this README (which is very similar).
```suggestion
StumpWM init file:
```